src/invidious/videos.cr

```diff
@@ -110,7 +110,7 @@ struct Video
         fmt["url"] = JSON::Any.new("#{fmt["url"]}#{DECRYPT_FUNCTION.decrypt_signature(fmt)}")
       end
 
-      fmt["url"] = JSON::Any.new("#{fmt["url"]}&host=#{URI.parse(fmt["url"].as_s).host}")
+      fmt["url"] = JSON::Any.new("#{fmt["url"]}&host=#{URI.parse(fmt["url"].as_s).host}&pot=#{CONFIG.po_token}")
```

Comment:
This should be behind a `if CONFIG.po_token` conditional, but I'll fix that when merging the two PRs.

Reply:
Yeah but an empty value for a query string won't really hurt.